rllib/core/rl_module/rl_module.py

```diff
@@ -407,3 +442,27 @@ def as_multi_agent(self) -> "MultiAgentRLModule":
         from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
 
         return MultiAgentRLModule({DEFAULT_POLICY_ID: self})
+
+    def __get_default_view_requirements(self):
+        obs_space = self.config.observation_space
```

Comment:
@kouroshHakha This assumes that RLModule has self.config.
It's not in the default base constructor though and we might want to think about this more.